### PR TITLE
[13.0][MIG] partner_identification

### PR DESCRIPTION
**Pull request by Laurent-Corron**

### CODE REVIEW

**Comment by mlaitinen** [comment]
@Laurent-Corron Please check the code coverage

**Review by ThomasBinsfeld — APPROVED** [review]

**Review by chrisdec — APPROVED** [review]
Functional tests

File: partner_identification/models/res_partner_id_number.py [hunk 1]
```diff
@@ -22,8 +22,7 @@ def validate_id_number(self):
     name = fields.Char(
         string="ID Number",
         required=True,
-        help="The ID itself. For example, Driver License number of this "
-        "person",
+        help="The ID itself. For example, Driver License number of this " "person",
```
Comment (NL66278) [inline partner_identification/models/res_partner_id_number.py]: You might merge the two strings (remove the double doublequotes between this and person)

**Review by NL66278 — APPROVED** [review]
:+1: One nag, but not blocking. Thanks for migrating this!

File: partner_identification/models/res_partner_id_number.py (hunk 1 above)
Comment (Laurent-Corron) [reply partner_identification/models/res_partner_id_number.py]: done! :)

**Review by ntsirintanis — APPROVED** [review]

**Comment by NL66278** [comment]
This failes due to a test in the partner_firstname module. For some reason when checking the name 'Foo bis', bis is not the firstname, but also not seen as part of the lastname. Anybody any idea?

**Comment by rousseldenis** [comment]
> This failes due to a test in the partner_firstname module. For some reason when checking the name 'Foo bis', bis is not the firstname, but also not seen as part of the lastname. Anybody any idea?

Maybe @lmignon as you worked on it.

@NL66278 You can just use bot merge command without major. If so, it will change version from 13.0.1.0.0 to 13.0.2.0.0

File: partner_identification/models/res_partner.py [hunk 2]
```diff
@@ -0,0 +1,163 @@
...
+        Args:
+            field_name (str): Name of field to set.
+            category_code (str): Category code of the Identification type.
+        """
+        for record in self:
+            id_numbers = record.id_numbers.filtered(
+                lambda r: r.category_id.code == category_code
+            )
+            if not id_numbers:
+                continue
```
Comment (ntsirintanis) [inline partner_identification/models/res_partner.py:57]: When we use this method as a compute method for a field, then things break if there's no `id_numbers`. We should do `record[field_name] = False` and then `continue`.

**Review by ntsirintanis — CHANGES_REQUESTED** [review]

**Pull request merged**